### ISSUE TITLE
Partial bundle for the treemap

### ISSUE THREAD

**Issue by edwinmp** [plotly/plotly.js#4949]
Not sure if it's a documentation issue, but there doesn't seem to be a partial bundle that supports the `treemap`

https://github.com/plotly/plotly.js/blob/master/dist/README.md#partial-bundles

**Comment by archmoj** [comment]
It is in [finance](https://github.com/plotly/plotly.js/blob/master/dist/README.md#plotlyjs-finance).

**Comment by edwinmp** [comment]
Great, thanks @archmoj

**Comment by urlsangel** [comment]
Hi @archmoj 

Thanks for the tip :-)

I've switched the module to https://github.com/plotly/plotly.js/blob/master/dist/README.md#plotlyjs-finance, but my test `treemap` isn't rendering, whereas it renders as expected using the full `plotly-latest` distribution.

### Treemap rendered with `finance` module: 
<img width="740" alt="Screenshot 2020-06-25 at 09 12 25" src="https://user-images.githubusercontent.com/560463/85682298-0f043280-b6c4-11ea-8472-0ddd9750363d.png">

### Treemap rendered with `plotly-latest`: 
<img width="723" alt="Screenshot 2020-06-25 at 09 14 28" src="https://user-images.githubusercontent.com/560463/85682533-4d99ed00-b6c4-11ea-8b39-f74b107c7ff7.png">

Here's the initial part of my data set, with the trace `type` set to `treemap`:

```
{
    "data": [
        {
            "meta": {
                "columnNames": {
                    "ids": "ID",
                    "labels": "Label",
                    "values": "Value",
                    "parents": "Parent"
                }
            },
            "mode": "markers",
            "name": "2015",
            "type": "treemap",
...
``` 

Do you know where I could find a working example of a `treemap` being rendered using the `finance` module so I could try to work out what's wrong?

Thanks!

**Comment by archmoj** [comment]
> It is in [finance](https://github.com/plotly/plotly.js/blob/master/dist/README.md#plotlyjs-finance).

Correction: there in no partial bundle for `treemap` & `sunburst` traces.

This issue is currently not on our roadmap in the coming months but we would happily accept a pull request if someone wants to implement it, or [we would accept sponsorship](https://plot.ly/products/consulting-and-oem/) to accelerate its development.

**Comment by urlsangel** [comment]
Thanks for the info @archmoj 👍😀

**Comment by archmoj** [comment]
This would be addressed by #5508.
